packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -50,7 +50,7 @@ interface ResultsPaginationEvent {
 const ResultsPagination: FC<ResultsPaginationProps> = ({
   page = 1,
   pageSizes = [10, 20, 30, 40, 50],
-  pageSize,
+  pageSize = 10,
```

Comment:
Would default to 5 if not set (for some reason), even though that wasn't one of the `pageSizes` specified. So I changed it to default to 10.